tests/backup/backup_test.go

```diff
@@ -175,19 +191,20 @@ var _ = Describe("{BasicBackupCreation}", func() {
 		})
 
 		Step("Creating rules for backup", func() {
-			log.InfoD("Creating pre rule for deployed apps")
-			pre_rule_status := CreateRuleForBackup("backup-pre-rule", "default", app_list, "pre", ps)
+			dash.Info("Creating pre rule for deployed apps")
```

Comment:
change it to log.InfoD ,which will push to both dash board and log file
please refer https://portworx.atlassian.net/wiki/spaces/INFRA/pages/2591817827/Framework+Integration+with+Aetos

Reply:
done